rust/worker/src/execution/operators/projection.rs

```diff
@@ -49,7 +49,7 @@ pub struct ProjectionInput {
     pub offset_ids: Vec<u32>,
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
fwiw this feels like a type that already exists in our code elsewhere and should not have been invented here.

Reply:
This is similar to `DataRecord` and `OperationRecord` in the `types` crate. `DataRecord` stores references, while `OperationRecord` has extra fields.